src/test/java/org/icgc/argo/program_service/grpc/GRpcServerRunnerIT.java

```diff
@@ -0,0 +1,24 @@
+package org.icgc.argo.program_service.grpc;
+
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.health.v1.HealthCheckRequest;
+import io.grpc.health.v1.HealthCheckResponse;
+import io.grpc.health.v1.HealthGrpc;
+import lombok.val;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class GRpcServerRunnerIT {
```

Comment:
can you use `Test` in the class name so that it makes it easier to visually filter when searching through classes.